src/transformStateWithClones.js

```diff
@@ -1,13 +1,37 @@
 'use strict';
 
+const ADD_ACTION = 'addProperties';
+const REMOVE_ACTION = 'removeProperties';
+const CLEAR_ACTION = 'clear';
+
 /**
  * @param {Object} state
  * @param {Object[]} actions
  *
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let temp = { ...state };
+  const states = [];
+
+  actions.forEach(action => {
```

Comment:
Would be better to use `for of` loop instead forEach, don't use forEach, it's a bad practice